### ISSUE TITLE
Some users can't receive coins in IAP

### ISSUE THREAD

**Issue by immartian** [Musicoin/app#9]
There are several users can't receive coins in their account after newly creating an account in the app, yet their money in Apple account would be deducted.

**Comment by immartian** [comment]
it's related to the backend, but the client should have protection. https://github.com/Musicoin/api/issues/22

**Comment by bengyles** [comment]
I propose we block the IAP when there is not wallet available in the app and show a message there is no wallet address available in the wallet screen for now. I think it's also a good idea to hide the "$Buy MUSIC" option as well when this issue occurs.

When we have a solution in the backend we can add a button to "solve" the wallet issue in case we need that. What do you think @immartian @riverfor ?

**Comment by immartian** [comment]
+1 
It can be a hotfix as step 1, right?  
so step 2, if I'm not wrong, is to give users an option to fix it by clicking a link("get a new wallet"), likely?

**Comment by bengyles** [comment]
@immartian Yes indeed, step 1 is a hotfix and can be pushed over the air. Step 2 can also be done over the air but needs some more development, we can add it as a hotfix too or include it in the next version. If we could just fix it in the backend without having to add a button or if they can just logout and login again to fix the problem that would be better ofcourse.

**Comment by immartian** [comment]
We need both of them implemented asap as IAP is the crucial line of us on user's satisfaction and our own income. Please have a hotfix first then integrate backend solution to have an inhere resolve.

**Comment by riverfor** [comment]
READY FOR TEST on t2.
@bengyles @immartian
